queries.yaml

```diff
@@ -9,6 +9,7 @@ all:
   - boundaries
   - transit
   - admin_areas
+  - contours
```

Comment:
This would be an optional layer for the mainline build... and a fork could put it into the `all` grouping.

Reply:
Updated according to your comment.